NEWS.rst

```diff
@@ -8,10 +8,46 @@ guake
 Release Summary
 ---------------
 
+Eliminated redundant terminal spawning on startup
+
+Security Issues
+---------------
+
+- Fixed security Issue: Exposure of sensitive function, malicious user can arbitrary command via an execute_command d-bus method. #1796 
+
+Translation Updates
+-------------------
+
+- Chinese Simplified (@kitty-panics)
+
+- Croatian (@milotype)
+
+- Dutch (@Vistaus)
+
+- German (@m1ga, @rMb93)
+
+- Indonesian (@rezaalmanda)
+
+- Polish (@piotrdrag)
+
+- Russian (@vantu5z)
+
+- Swedish (@MorganAntonsson)
+
+- Turkish (@Draconis-25, @ersen0)
+
+3.8.2.0rc1
```

Comment:
even if the curernt release notes generator does not it, can we merge what it in the RCs and the current release?

Reply:
We can do that by either not tagging RCs, removing the RC tag which may break some history, or patching the news file every time we generate it. Alternately, if you don't like seeing RCs in the news log, you can [not tag them on pulls that were intended to be releases](https://github.com/Guake/guake/pull/2012)

Reply:
I have what I describe at work: RCs are Ignored in the diff. So yes they can be removed, but they help test changes in the release process.